etc/afpd/directory.c

```diff
@@ -981,8 +1232,10 @@ struct path *cname(struct vol *vol, struct dir *dir, char **cpath)
     uint16_t   len16;
     int         size = 0;
     int         toUTF8 = 0;
+    int         original_len =
+        0;  /* Save original len before while loop modifies it */
```

Comment:
another example of astyle adding an unfortunate line break with a long trailing comment!

put comment above the line or remove it